src/Http/src/Config/HttpConfig.php

```diff
@@ -51,4 +52,12 @@ public function getMiddleware(): array
     {
         return $this->config['middleware'] ?? $this->config['middlewares'];
     }
+
+    /**
+     * If chunkSize isn't provided - using default values
+     */
+    public function getChunkSize(): ?int
+    {
+        return !\is_null($this->config['chunkSize']) ? (int) $this->config['chunkSize'] : null;
```

Comment:
returned value should be positive int or null

Reply:
Added a check that the value is greater than 0